next/components/Molecules/GalleryBanner.tsx

```diff
@@ -1,114 +1,14 @@
-import { ComponentLocalityPartsGalleryParts } from '@services/graphql'
-import { useTranslation } from 'next-i18next'
+import { UploadImageEntityFragment } from '@services/graphql'
 import React from 'react'
 
-import GalleryModal from './GalleryModal'
+import ImageGallery from './ImageGallery'
 
 export interface PageProps {
-  gallery?: Array<ComponentLocalityPartsGalleryParts | null | undefined>
+  gallery?: UploadImageEntityFragment[]
```

Comment:
Shouldn'd we call this prop `images`?
I also noticed that props are named "PageProps" and they are exported for some reason, I don't think we need to export it.
For consistency, we also try to change props interfaces to types.